Classes/Hooks/UsageProcessAfterFinishHook.php

```diff
@@ -66,4 +70,37 @@ private function getLanguageService(): LanguageService
     {
         return $GLOBALS['LANG'];
     }
+
+    /**
+     * Make large API limits easier to read
+     *
+     * @param int $number Any large integer - 5000000
+     * @return string Formated, better readable string variant of the integer - 5.000.000
+     */
+    private function formatNumber(int $number): string
+    {
+        return number_format($number, 0, ',', '.');
```

Comment:
Using the `NumberFormatter` would be nicer, however that requires php `intl` which is only hard required by the TYPO3 core since v12.

Reply:
Yes indeed, since this would take local langauge settings into account. But as you said, the requrements does not exist in TYPO3 11 and adding this to the package only for this one number format seems to be a disproportionate burden for integrators. Suggestion: Merge this format for now, keep a migration to NumberFormatter:: on a short list for your next version?

Reply:
Changed this to a fallback solution using the Symfony polyfill, if set and moved tu UsageService for better maintenance

Reply:
Talked team internal about it. Fine for me.